docs/stransmit.md

```diff
@@ -80,55 +99,115 @@ Be careful with options being specified together with having standard
 output as output URI - some of them are not allowed as the extra output
 controlled by options might interfere with the data output.
 
+
 Medium: UDP
 -----------
 
 UDP can only be used in listening mode for reading, and in calling mode
-for writing. Therefore, when UDP is your \<input-uri\>, you usually
-specify the local port, e.g.:
+for writing. The multicast specification is also possible. The specification
+and meaning of the fields in the URI depend on the mode.
+
+The **PORT** part is always mandatory and it designates the port number
+for either the target host or to be bound to read from.
```

Comment:
he **PORT** part is always mandatory and it designates either the port number
for the target host, or the port number to be bound to read from.